src/PHPUnit/HttpMockFacade.php

```diff
@@ -84,4 +90,60 @@ public function each(callable $callback)
     {
         $callback($this);
     }
+
+    /**
+     * Get error output (if exists)
+     *
+     * @return string
+     */
+    public function getErrorOutput()
+    {
+        return static::cleanErrorOutput($this->server->getIncrementalErrorOutput());
+    }
```

Comment:
I think this method should be moved to InterNations\Component\HttpMock\Server. The other classes should not be aware of "logs needs to be filtered". A separate error filter class seems a little too much for me but I will do this if you want, you are mainaining it ;-)